charts/datadog/templates/agent-cilium-network-policy.yaml

```diff
@@ -152,11 +152,11 @@ specs:
               - port: "{{ $.Values.datadog.apm.port }}"
                 protocol: TCP
 {{- end }}
-# The agents are susceptible to connect to any pod
-# that would be annotated with auto-discovery annotations.
+# The agents are susceptible to an issue connecting to any pod that
+# is annotated with auto-discovery annotations.
```

Comment:
adjusted the wording on these 4 again; let me know if this doesn't accurately describe the issue and / or needs additional improvements.